### PR TITLE
[CHE-147] Restyle Buttons For Consistency

### PR DESCRIPTION
**Pull request by Mnelson98**
### Description

As part of [CHE-113](https://codehammers.atlassian.net/browse/CHE-113?atlOrigin=eyJpIjoiNDYwNGY0YTEwODM2NGE1NjgwMjBiNjlkYjJmMjg0N2QiLCJwIjoiaiJ9) story "profiles improvement", this PR restyles the buttons for the "edit profile" page and account button in the "header".

### Jira Task

[CHE-147](https://codehammers.atlassian.net/browse/CHE-147?atlOrigin=eyJpIjoiNDI4MmViOWZlZGNkNDkzNGI0MDYxZTRmMTU0OGU0MjAiLCJwIjoiaiJ9) 

### Testing Instructions

1. Make sure all branches are up to date and you are working from feature branch
2. Log in and spin of the dev environment
3. Review and test all buttons 

#### All Team Members

- [x] I added a descriptive title to this PR.
- [x] I filled out the **Description**, **Jira Task**, and **Testing Instructions** sections above.
- [ ] I added or updated [Jest unit tests]for any changes to components, server-side controllers, etc.
- [x] I ran `npm run docker-test` in my local environment to check that this PR passes all unit tests.
- [x] I did a quick check to make sure my code changes follow the recomended style guide.

### Additional Notes, Images, etc.

N/A

[CHE-113]: https://codehammers.atlassian.net/browse/CHE-113?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CHE-147]: https://codehammers.atlassian.net/browse/CHE-147?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: client/src/pages/EditProfilePage/EditProfilePage.tsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ const EditProfilePage = () => {
   useEffect(() => {
     if (userID) dispatch(fetchUserProfile(userID as string));
   }, [dispatch]);
-
+  // Micah was here
```
Comment (jimpulsee) [inline client/src/pages/EditProfilePage/EditProfilePage.tsx]: Can remove

File: client/src/components/Header/Header.tsx [hunk 2]
```diff
@@ -77,7 +77,7 @@ const Header = (): JSX.Element => {
       <div className="relative">
         <button
           onClick={() => setShowDropdown(!showDropdown)}
-          className="bg-gray-700 hover:bg-gray-800 font-bold py-2 px-4 rounded"
+          className="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded"
```
Comment (jimpulsee) [inline client/src/components/Header/Header.tsx]: Please have a look at the style guide and order CSS properties / class names alphabetically

Comment (jimpulsee) [reply client/src/components/Header/Header.tsx]: A good practice to gradually decrease tech debt is to refactor the files involving your new features with each PR!

**Review by jimpulsee — CHANGES_REQUESTED** [review]

File: client/src/components/Header/Header.tsx (hunk 2 above)
Comment (brok3turtl3) [reply client/src/components/Header/Header.tsx]: Good catch @JimmyTran48. To be fair Micah probably used my buttons as examples and they were implemented before our style guide choice..

**Review by brok3turtl3 — APPROVED** [review]
Great job @Mnelson98. Refactor looks good and the buttons look awesome!

**Review by jimpulsee — APPROVED** [review]
LGTM

**Pull request merged**